<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemDialog</class>
 <widget class="QDialog" name="NewItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление нового изделия</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="MCchipName"/>
     </item>
     <item row="6" column="1">
      <widget class="QDateTimeEdit" name="MCdateTimeEnd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy H:mm (dddd)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="MCdateTimeStart">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy H:mm (dddd)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="MCmapLabel">
       <property name="text">
        <string>Маршрутная карта</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="MCCommentLabel">
       <property name="text">
        <string>Комментарий</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="MCidLabel">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="MCduration">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="MCdurationLabel">
       <property name="text">
        <string>Длительность испытаний, ч</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="MCdateTimeEndLabel">
       <property name="text">
        <string>Окончание испытаний</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="MCdateTimeStartLabel">
       <property name="text">
        <string>Начало испытаний</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="MCmap"/>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="MCid">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="MCcountLabel">
       <property name="text">
        <string>Количество, шт</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="MCcount"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="MCdevice"/>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="MCtestIsDone">
       <property name="text">
        <string>проведены</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="MCtestIsDoneLabel">
       <property name="text">
        <string>Испытания</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPlainTextEdit" name="MCcomment">
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="MCchaipNameLabel">
       <property name="text">
        <string>Изделие</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" rowspan="2">
      <widget class="QLabel" name="MCdeviceLabel">
       <property name="text">
        <string>Оборудование</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="setCurrentDateTimeButton">
       <property name="text">
        <string>Установить текущее датувремя</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>316</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>MCchipName</tabstop>
  <tabstop>MCmap</tabstop>
  <tabstop>MCcount</tabstop>
  <tabstop>MCdateTimeStart</tabstop>
  <tabstop>MCduration</tabstop>
  <tabstop>MCdateTimeEnd</tabstop>
  <tabstop>MCdevice</tabstop>
  <tabstop>MCcomment</tabstop>
  <tabstop>MCtestIsDone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
